standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -213,6 +213,19 @@
   //todo: make these like OperationType and remove above char constatns
   enum TxnStatus {OPEN, ABORTED, COMMITTED, UNKNOWN}
 
+  public enum TxnType {
+    REPL_CREATED(0), READ_ONLY(1), UNKNOWN(2);
```

Comment:
For TxnState, UNKNOWN is used to specify an invalid state of txn. So, thought it would make sense to use something else to mean Read-Write non-REPL created txns instead of UNKNOWN.

Reply:
need change